lib/streamlit/elements/arrow_altair.py

```diff
@@ -290,15 +281,15 @@ def _arrow_bar_chart(
     @gather_metrics("_arrow_altair_chart")
     def _arrow_altair_chart(
         self,
-        altair_chart: Chart,
+        altair_chart: "Chart",
```

Comment:
Nit: I think you can avoid the quotes for this type annotation if you add a `from __future__ import annotations` at the top of the file (even though the import is conditional on the `TYPE_CHECKING` flag)

Reply:
Oh, I didn't know. I will update this later :)